addons/sales_team/tests/test_sales_team.py

```diff
@@ -204,6 +205,18 @@ def test_team_memberships(self):
         with self.assertRaises(exceptions.UserError):
             team_c2.write({'company_id': self.company_2.id})
 
+    def test_user_allowed_to_join_team_in_company_ids(self):
```

Comment:
Can't we include it in test_team_members so that it covers various company use cases ?